views/article.html

```diff
@@ -109,6 +110,17 @@ <h2 class="article-standfirst">{{displaySummary}}</h2>
 
 						<div amp-access="access OR {{freeArticle}}" amp-access-hide>
 							{{{mainImageHtml}}}
+
+							<div class="article-social-buttons">
+								<amp-social-share width="24" height="24" type="twitter" data-param-url="{{shareUrl}}"></amp-social-share>
+								<amp-social-share width="24" height="24" type="facebook"
+									data-param-href="{{shareUrl}}"
+									data-param-app_id="{{facebookAppId}}"></amp-social-share>
+								<amp-social-share width="24" height="24" type="linkedin" data-param-url="{{shareUrl}}"></amp-social-share>
+								<amp-social-share width="24" height="24" type="whatsapp"
+									data-share-endpoint="whatsapp://send"
+									data-param-text="Check out this article: TITLE - {{shareUrl}}"></amp-social-share>
```

Comment:
That's an AMP variable substitution https://github.com/ampproject/amphtml/blob/master/extensions/amp-social-share/amp-social-share.md#un-configured-providers